src/gromacs/nblib/util.h

```diff
@@ -231,6 +236,20 @@ auto binary_fold(Op&& op, First&& first, Ts&&... args)
     return op(first, binary_fold(op, args...));
 }
 
+template<class... Args>
+std::string formatString(std::string fmt, Args... args)
+{
+    std::ostringstream os;
+    std::string        delimiter = "{}";
+
+    __attribute__((unused)) std::initializer_list<int> unused{
```

Comment:
Nice. Is it portable though?

Reply:
Probably only works for gcc and clang https://nelkinda.com/blog/suppress-warnings-in-gcc-and-clang/